### PR TITLE
refactor: move derived token to store

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Move `$token` from `token.derived` to  a new module `token.store` as we are about to transform it into a writable store.

# Changes

- Only move. No functional changes.

### CODE REVIEW

File: src/frontend/src/eth/components/send/SendModal.svelte [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <script lang="ts">
 	import SendTokenModal from '$eth/components/send/SendTokenModal.svelte';
 	import SendTokenContext from '$eth/components/send/SendTokenContext.svelte';
-	import { token } from '$lib/derived/token.derived';
+
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/eth/components/send/SendModal.svelte:4]: It is a small thing, but may you please remove this empty paragraph?

File: src/frontend/src/icp/components/transactions/IcTransactionsCkEthereumListeners.svelte [hunk 2]
```diff
@@ -5,7 +5,8 @@
 	import { ckEthereumNativeToken } from '$icp-eth/derived/cketh.derived';
 	import type { CanisterIdText } from '$lib/types/canister';
 	import type { OptionIcCkToken } from '$icp/types/ic';
-	import { token } from '$lib/derived/token.derived';
+
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/icp/components/transactions/IcTransactionsCkEthereumListeners.svelte:8]: ditto

File: src/frontend/src/icp/components/transactions/IcTransactionsSkeletons.svelte [hunk 3]
```diff
@@ -3,7 +3,8 @@
 	import SkeletonCards from '$lib/components/ui/SkeletonCards.svelte';
 	import { nonNullish } from '@dfinity/utils';
 	import { icTransactionsStore } from '$icp/stores/ic-transactions.store';
-	import { token } from '$lib/derived/token.derived';
+
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/icp/components/transactions/IcTransactionsSkeletons.svelte:6]: ditto

File: src/frontend/src/routes/(app)/transactions/+layout.svelte [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <script lang="ts">
 	import Listener from '$lib/components/core/Listener.svelte';
-	import { token } from '$lib/derived/token.derived';
+
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/routes/(app)/transactions/+layout.svelte:3]: ditto

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
Looks good!

Only small things that I pointed out in the changes: there are a few empty paragraphs, maybe you want to remove them before merge.

File: src/frontend/src/eth/components/send/SendModal.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/eth/components/send/SendModal.svelte]: Definitely, good catch!

File: src/frontend/src/icp/components/transactions/IcTransactionsCkEthereumListeners.svelte (hunk 2 above)
Comment (peterpeterparker) [reply src/frontend/src/icp/components/transactions/IcTransactionsCkEthereumListeners.svelte:8]: Webstorm seems to have introduced those...Thanks!

**Pull request merged**